README.md

```diff
@@ -54,9 +54,12 @@ in the monorepo.
 
 Each package should define the following scripts in `package.json`:
 
-- `build`: Builds the package as preparation for publishing or use by its dependencies.
 - `check`: Lints, formats, and (if applicable) type-checks the package.
 - `check:fix`: Lints, formats, type-checks, and fixes issues where possible.
+- `test`: Runs tests for the package.
+
+If a package has a build step, it should also define:
+- `build`: Builds the package as preparation for publishing or use by its dependencies.
```

Comment:
Adding `test` as a required script for new changes, and clarifying that packages only need a `build` script if they have build output.